hummingbot/connector/exchange/bittrex/bittrex_exchange.py

```diff
@@ -0,0 +1,403 @@
+                percent_token = tracked_order.quote_asset
+                fee = TradeFeeBase.new_spot_fee(
+                    fee_schema=self.trade_fee_schema(),
+                    trade_type=tracked_order.trade_type,
+                    percent_token=percent_token,
+                    flat_fees=[TokenAmount(amount=Decimal(execution_event["commission"]),
+                                           token=percent_token)]
+                )
+                trade_update = TradeUpdate(
+                    trade_id=execution_event["id"],
+                    client_order_id=tracked_order.client_order_id,
+                    exchange_order_id=order_id,
+                    trading_pair=tracked_order.trading_pair,
+                    fee=fee,
+                    fill_base_amount=Decimal(execution_event["quantity"]),
+                    fill_quote_amount=Decimal(execution_event["quantity"]) * Decimal(execution_event["rate"]),
+                    fill_price=Decimal(execution_event["rate"]),
+                    fill_timestamp=_get_timestamp(execution_event["executedAt"]),
+                )
+                self._order_tracker.process_trade_update(trade_update)
+
+    def _get_fee(self,
+                 base_currency: str,
+                 quote_currency: str,
+                 order_type: OrderType,
+                 order_side: TradeType,
+                 amount: Decimal,
+                 price: Decimal = s_decimal_NaN,
+                 is_maker: Optional[bool] = None) -> TradeFeeBase:
+        is_maker = is_maker or (order_type is OrderType.LIMIT_MAKER)
+        fee = build_trade_fee(
+            self.name,
+            is_maker,
+            base_currency=base_currency,
+            quote_currency=quote_currency,
+            order_type=order_type,
+            order_side=order_side,
+            amount=amount,
+            price=price,
+        )
+        return fee
+
+    def _get_new_client_order_id(self,
+                                 is_buy: bool,
+                                 trading_pair: str,
+                                 hbot_order_id_prefix: str,
+                                 max_id_len: int
+                                 ):
+        return str(uuid.uuid4())
+
+    def _initialize_trading_pair_symbols_from_exchange_info(self, exchange_info: List):
+        mapping = bidict()
+        for symbol_data in filter(bittrex_utils.is_exchange_information_valid, exchange_info):
+            mapping[symbol_data["symbol"]] = combine_to_hb_trading_pair(base=symbol_data["baseCurrencySymbol"],
+                                                                        quote=symbol_data["quoteCurrencySymbol"])
+        self._set_trading_pair_symbol_map(mapping)
+
+    async def _get_last_traded_price(self, trading_pair: str) -> float:
+        exchange_symbol = await self.exchange_symbol_associated_to_pair(trading_pair=trading_pair)
+        params = {
+            "marketSymbol": exchange_symbol
+        }
```

Comment:
According to the documentation this endpoint does not receive parameters (the marketSymbol is part of the URL).
https://bittrex.github.io/api/v3#operation--markets--marketSymbol--ticker-get

Reply:
Removed the redundancy